docs/api-reference/stats-widget/stats-widget.md

```diff
@@ -8,42 +8,75 @@ Create a `StatsWidget` HTML element to display tracked `Stats`. Each `Stat` can
 be associated with a `formatter` that indicates how it should be displayed.
 
 ```js
+import React, {Component} from 'react';
+import {Stats} from 'probe.gl';
 import StatsWidget from '@probe.gl/stats-widget';
 
-const stats = new Stats({id: 'My Stats'});
-const counter = stats.get('Counter');
-const statsWidget = new StatsWidget(stats);
-
-for (let i = 0; i < 100; i++) {
-  counter.addCount(i);
+class App extends Component {
+  componentDidMount() {
+    this._stats = new Stats({
+      id: 'My Stats'
+    });
+    
+    this._statsWidget = new StatsWidget({
+      stats: this._stats,
+      container: this._containerRef
+    });
+
+    this.setState({intervalId: setInterval(this._update, 300)});
+  }
+
+  componentWillUnmount() {
+    // use intervalId from the state to clear the interval
+    clearInterval(this.state.intervalId);
+  }
+
+  _update() {
+    // create a stat with name and type
+    const counter = this._stats.get('Counter', 'count');
+    counter.incrementCount();
+    this._statsWidget.update();  
+  }
+
+  render() {
+    return (<div ref={_ => this._containerRef = _}/>);
+  }
 }
-statsWidget.update();
+
 ```
 
 ## Methods
 
 ### constructor
 
-`new StatsWidget(stats, opts)`
+`new StatsWidget(props)`
```

Comment:
This is a breaking change, we'll have to update in many places after we publish and upgrade other examples. 

To avoid that, should we keep the old signature and just allow the app to pass `null` as first argument?

Reply:
I'll keep the constructor as `(stats, options)` for now, but I would like to make this change in the next major release.
And I checked with our other frameworks, mostly used in website directory.